--- zoo_sample.cfg	2012-09-30 19:53:31.000000000 +0200
+++ zoo.cfg	2013-09-10 01:00:58.000000000 +0200
@@ -9,7 +9,7 @@
 # the directory where the snapshot is stored.
 # do not use /tmp for storage, /tmp here is just 
 # example sakes.
-dataDir=/tmp/zookeeper
+dataDir=${zookeeper.datafolder_}
 # the port at which the clients will connect
 clientPort=2181
 #
@@ -19,7 +19,7 @@
 # http://zookeeper.apache.org/doc/current/zookeeperAdmin.html#sc_maintenance
 #
 # The number of snapshots to retain in dataDir
-#autopurge.snapRetainCount=3
+autopurge.snapRetainCount=3
 # Purge task interval in hours
 # Set to "0" to disable auto purge feature
-#autopurge.purgeInterval=1